internal/controller/operator/factory/finalize/common.go

```diff
@@ -65,11 +65,11 @@ func RemoveFinalizer(ctx context.Context, rclient client.Client, instance client
 	})
 }
 
-func removeFinalizeObjByName(ctx context.Context, rclient client.Client, obj client.Object, name, ns string) error {
-	return removeFinalizeObjByNameWithOwnerReference(ctx, rclient, obj, name, ns, true)
+func RemoveFinalizeObjByName(ctx context.Context, rclient client.Client, obj client.Object, name, ns string) error {
+	return RemoveFinalizeObjByNameWithOwnerReference(ctx, rclient, obj, name, ns, true)
 }
 
-func removeFinalizeObjByNameWithOwnerReference(ctx context.Context, rclient client.Client, obj client.Object, name, ns string, keepOwnerReference bool) error {
+func RemoveFinalizeObjByNameWithOwnerReference(ctx context.Context, rclient client.Client, obj client.Object, name, ns string, keepOwnerReference bool) error {
```

Comment:
Proper way of such refactoring is the following:
* introduce new public package method with proper comment
* keep exist private method and use it with public
* use new method for the new business logic ( keep previous logic as is).
* after changes merged - replace private method with new public at separate PR.

 This way keeps changes non-intrusive and easy to review.